testing/surf/APKBUILD

```diff
@@ -0,0 +1,29 @@
+# Contributor: Orson Teodoro <orsonteodoro@hotmail.com>
+# Maintainer: Orson Teodoro <orsonteodoro@hotmail.com>
+pkgname=surf
+pkgver=2.0
+pkgrel=0
+pkgdesc="surf is a simple web browser based on WebKit2/GTK+."
+url="https://surf.suckless.org/"
+arch="all"
+license="MIT"
+depends="xprop"
+makedepends="webkit2gtk-dev"
+options="!check"
+source="$pkgname-$pkgver.tar.bz2::https://git.suckless.org/$pkgname/snapshot/$pkgname-$pkgver.tar.bz2"
+builddir="$srcdir/"$pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	make
+}
+
+package() {
+	cd "$builddir"
+	make DESTDIR="$pkgdir" PREFIX="/usr" install
+	gzip "$pkgdir"/usr/share/man/man1/surf.1
```

Comment:
You don't need to manually compress the man page, this is done by the doc subpackage function.

Reply:
For some reason it complains.  Do I need to explicitly call $pkgname-doc to compress it automatically?

Reply:
You need to define a list of subpackages using the `subpackages` variable:

```
subpackages="$pkgname-doc"
```